README.md

```diff
@@ -4,5 +4,29 @@ AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/fxhto3omtehio
 
 Travis:   [![Travis](https://travis-ci.org/aspnet/dnx-watch.svg?branch=dev)](https://travis-ci.org/aspnet/dnx-watch)
 
-This project is part of ASP.NET 5. You can find samples, documentation and getting started instructions for ASP.NET 5 at the [Home](https://github.com/aspnet/home) repo.
+With `dnx --watch`, when a file changed, dnx would exit and you had to restart it manually.
+`dnx-watch` is a file watcher for `dnx`. It replaces `dnx --watch` and it restarts the process.
+
+### How To Install
+`dnu commands install Microsoft.Dnx.Watcher`
+
+You might need to pass `--fallbacksource https://myget.org/F/aspnetvnext/api/v3/index.json` if you don't have that feed in the global NuGet config.
```

Comment:
Not exactly - I'd say more accurately that "to install the latest unstable release add the following parameter to the command".